server/internal/server/models.go

```diff
@@ -204,16 +206,18 @@ func (s *WS) RegisterModel(
 		return nil, status.Error(codes.InvalidArgument, "project_id is required")
 	}
 
-	id, err := s.genenerateModelID(req.BaseModel, req.Suffix)
-	if err != nil {
-		return nil, status.Errorf(codes.Internal, "generate model ID: %s", err)
+	id := req.Id
```

Comment:
Could you make this RPC return the `AlreadyExists` error if the `id` field is specified and there is already a model of the same ID?

Reply:
This field allows a user to name the model arbitrarily. The name of the model downloaded from HF can be in any format, we don't have to converted its name into "base-model-name.suffix" format.  

This ID would be also the model name to be used in inference request.